src/cascadia/TerminalApp/TerminalAppLib.vcxproj

```diff
@@ -53,7 +53,7 @@
     <Page Include="TabRowControl.xaml">
       <SubType>Designer</SubType>
     </Page>
-        <Page Include="TabHeaderControl.xaml">
+    <Page Include="TabHeaderControl.xaml">

```

Comment:
I added the two new files using VT instead of adding them by hand. Hence all these changes.
But I looked at them and thought that it's actually an appropriate cleanup and left it in this PR.